src/Pages/Category.js

```diff
@@ -0,0 +1,117 @@
+import { NavLink } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import React, { useEffect, useState } from 'react';
+import { ImSearch } from 'react-icons/im';
+import { FaArrowRight } from 'react-icons/fa';
+import { getPhotos, photoId, searchPhoto } from '../Redux/PhotosSlice';
+import './Category.css';
+
+const Category = () => {
+  const dispatch = useDispatch();
+
+  const photos = useSelector((state) => state.Photos.data);
+  const apiParameters = useSelector((state) => state.Photos.parameters);
+  const [parameters, setParameters] = useState(apiParameters);
+
+  /* console.log(photos); */
+  /* console.log(parameters); */
```

Comment:
- [ ] As mentioned in the javascript bect practices we 
  - **Do not commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data be visible to 
    anyone using the browser tools to look at your website.**
  - **Do not commit old pieces of code as inline comments. They will make your project look messy.**

Please remove commented code and console log statements from your all over your code